frontend/pages/donations/error.jsx

```diff
@@ -0,0 +1,23 @@
+import Link from "next/link";
+import React from "react";
+import Layout from "../../components/Layout";
+
+export default function Error () {
```

Comment:
It would be nicer to conditionally render error content on the /donations/make page, rather than using a separate page.  I'll add this to https://modularhistory.atlassian.net/browse/MH-203.